api/src/main/java/org/openmrs/PatientIdentifier.java

```diff
@@ -36,7 +37,7 @@
  * @see org.openmrs.PatientIdentifierType
  */
 @Indexed
-public class PatientIdentifier extends BaseChangeableOpenmrsData implements java.io.Serializable, Cloneable, Comparable<PatientIdentifier> {
+public class PatientIdentifier extends BaseChangeableOpenmrsData implements java.io.Serializable, Cloneable, Comparable<PatientIdentifier>, PersonPatientSharable<PatientIdentifier> {
```

Comment:
Thanks alot for working on this @gracebish What did you intend to do there,, by adding PersonPatientSharable package

Reply:
I created an interface with methods i used in the files i changed